app/preprints/-components/submit/metadata/component.ts

```diff
@@ -8,11 +14,49 @@ interface MetadataArgs {
     manager: PreprintStateMachine;
 }
 
+interface MetadataForm {
+    publicationDoi: string;
+    publicationYear: number;
+    publicationCitation: string;
+}
+
+const MetadataFormValidation: ValidationObject<MetadataForm> = {
+    publicationDoi: validateFormat({
+        allowBlank: true,
+        allowNone: true,
+        ignoreBlank: true,
+        regex: DOIRegex,
+        type: 'invalid_doi',
+    }),
+    publicationYear: validateFormat({
+        allowBlank: true,
+        allowNone: true,
+        ignoreBlank: true,
+        regex: /^((?!(0))[0-9]{4})$/,
+        type: 'year_format',
+    }),
+    publicationCitation: validatePresence({
```

Comment:
If this `publicationCitation` is something that is needed (see prior comment), you'll need to add a `type:` field to this `validatePresence`. The `type` field is what the FormControl component uses under the hood to try to find the proper validation message (e.g. for `type: 'invalid_doi'`, it will show `{{t 'validationErrors.invalid_doi'}}`